### ISSUE TITLE
MacBook Pro 9.2 (13-inch, Mid 2012) - Bluetooth don't work.

### ISSUE THREAD

**Issue by diparrag** [BenSova/Patched-Sur#657]
### MacBook Pro 9.2 (13-inch, Mid 2012) - Bluetooth don't work.

1. Not enabled (**Figure 01**) 
2. Not appear in System Preferences (**Figure 02**).
3. Not available (**Figure 03**).

> **Figure 01 - Bluetooth is not enabled**
> ![image](https://user-images.githubusercontent.com/10927383/124432448-41188d80-dd37-11eb-9006-029119efa326.png)

> **Figure 02 - Bluetooth don't appear in System Preferences**
> ![image](https://user-images.githubusercontent.com/10927383/124433705-ac169400-dd38-11eb-82d6-ac83753186ff.png)

> **Figure 03 - Bluetooth is not available**
> ![image](https://user-images.githubusercontent.com/10927383/124432675-80df7500-dd37-11eb-9de7-31542cf3b0d6.png)

> Figura 04 - Patched Sur v0.2.1
> ![image](https://user-images.githubusercontent.com/10927383/124434329-5db5c500-dd39-11eb-959d-c7986d40d4e8.png)

> Figura 05 - macOS Big Sur v11.4
> ![image](https://user-images.githubusercontent.com/10927383/124434583-a8374180-dd39-11eb-9fed-ed2a526449c1.png)

**Comment by ozziepadilla** [comment]
Did you run Patchet Kexts after macOS Big Sur installation finish? in the second picture there is the option Patchet Kexts , get in there and click on it then let it finish and when you see a green button says “Restart to finish” don’t click on it just restart manually your Mac and check your Bluetooth will works

**Comment by diparrag** [comment]
> Did you run Patchet Kexts after macOS Big Sur installation finished?i n the second picture that you sent there is the option Patchet Kexts , get in there and click on it then let it finish and when you see a green button says “Restart to finish” don’t click on it just restart manually your Mac and check your Bluetooth if works

Yes, I did run patched kexts and restart manually when "restart to finish" appear, but Bluetooth still don't working.

**Comment by ozziepadilla** [comment]
Did the wifi works or not?

**Comment by diparrag** [comment]
> Did the wifi works or not?

Wifi is working ok.

**Comment by ozziepadilla** [comment]
Try this , turn on/off the wifi toggle, reboot your Mac, the wireless card it does wifi network conectividad , Bluetooth and Airdrop, seems weird that after Patchet Kexts only Bluetooth doesn’t work so try that trick

**Comment by diparrag** [comment]
> Try this , turn on/off the wifi toggle, reboot your Mac, the wireless card it does wifi network conectividad , Bluetooth and Airdrop, seems weird that after Patchet Kexts only Bluetooth doesn’t work so try that trick

I followed the steps that you mentioned but Bluetooth still don't working.

**Comment by Monkiey** [comment]
> > Try this , turn on/off the wifi toggle, reboot your Mac, the wireless card it does wifi network conectividad , Bluetooth and Airdrop, seems weird that after Patchet Kexts only Bluetooth doesn’t work so try that trick
> 
> I followed the steps that you mentioned but Bluetooth still don't working.
 Hi, So to clarify your Bluetooth was working fine on the previous version of macOS before you updated to 11.4 right?

**Comment by diparrag** [comment]
Yes, that's right.

**Comment by diparrag** [comment]
> > Try this , turn on/off the wifi toggle, reboot your Mac, the wireless card it does wifi network conectividad , Bluetooth and Airdrop, seems weird that after Patchet Kexts only Bluetooth doesn’t work so try that trick
> 
> I followed the steps that you mentioned but Bluetooth still don't working.
> Hi, So to clarify your Bluetooth was working fine on the previous version of macOS big sur or were you upgrading from Catalina before you updated to 11.4 right?

Yes, that's right.

**Comment by Monkiey** [comment]
> > > Try this , turn on/off the wifi toggle, reboot your Mac, the wireless card it does wifi network conectividad , Bluetooth and Airdrop, seems weird that after Patchet Kexts only Bluetooth doesn’t work so try that trick
> > 
> > 
> > I followed the steps that you mentioned but Bluetooth still don't working.
> > Hi, So to clarify your Bluetooth was working fine on the previous version of macOS big sur or were you upgrading from Catalina before you updated to 11.4 right?
> 
> Yes, that's right.

okay did you update maco big sur using the post install updater or the usb stick?

**Comment by diparrag** [comment]
> > > > Try this , turn on/off the wifi toggle, reboot your Mac, the wireless card it does wifi network conectividad , Bluetooth and Airdrop, seems weird that after Patchet Kexts only Bluetooth doesn’t work so try that trick
> > > 
> > > 
> > > I followed the steps that you mentioned but Bluetooth still don't working.
> > > Hi, So to clarify your Bluetooth was working fine on the previous version of macOS big sur or were you upgrading from Catalina before you updated to 11.4 right?
> > 
> > 
> > Yes, that's right.
> 
> okay did you update macos big sur using the post-install updater or the usb stick?

I used the post-install updater.

**Comment by Monkiey** [comment]
I would try the USB method with 11.4 and reinstall macos
